torchvision/prototype/features/_keypoint.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, Any, Tuple, Optional, Sequence
+
+import torch
+
+from ._feature import Feature, DEFAULT
+
+
+class KeyPoint(Feature):
+    descriptions: Optional[Sequence[str]]
```

Comment:
I believe `Keypoint` should also know about the image sizes on which they are mapped to. Otherwise we won't be able to perform operations such as `flip`.

Reply:
Good point, will add that.